### PR TITLE
Fix some linter warnings, part 2

### PR DESCRIPTION
**Pull request by matrixik**
Related #191

### CODE REVIEW

**Comment by matrixik** [comment]
Max 5 containers at the same time reached.

**Comment by timothyb89** [comment]
They will need to be rebuilt to pull in the new changes, but I don't know how urgent pushing new images out for most of this is given it's mostly code quality vs functional changes - if nothing else they'll get picked up the next time we release.

Feel free to bump versions and go crazy with `!push` in follow-up PRs if desired, though.

File: kafka-init/wait-for.sh [hunk 1]
```diff
@@ -54,9 +54,9 @@ wait_for_wrapper()
 {
     # In order to support SIGINT during timeout: http://unix.stackexchange.com/a/57692
     if [[ $QUIET -eq 1 ]]; then
-        timeout $BUSYTIMEFLAG $TIMEOUT $0 --quiet --child --host=$HOST --port=$PORT --timeout=$TIMEOUT &
+        timeout "$BUSYTIMEFLAG" "$TIMEOUT" $0 --quiet" --child --host="$HOST" --port="$PORT" --timeout="$TIMEOUT" &
```
Comment (digitalfishpond) [inline kafka-init/wait-for.sh]: Erroneous double quote after `--quiet`

Comment (matrixik) [reply kafka-init/wait-for.sh]: Done

**Comment by digitalfishpond** [comment]
I believe this is ready to merge by someone with appropriate rights :) @kornicameister @matrixik

**Pull request merged**